amounts.md

```diff
@@ -0,0 +1,341 @@
+There are 1 occurrences of 0.00110742 BTC output.
+There are 39 occurrences of 0.00131072 BTC output.
+There are 1 occurrences of 0.00134146 BTC output.
+There are 1 occurrences of 0.00141019 BTC output.
+There are 1 occurrences of 0.00181787 BTC output.
+There are 1 occurrences of 0.00230577 BTC output.
+There are 1 occurrences of 0.00235743 BTC output.
+There are 39 occurrences of 0.00262144 BTC output.
+There are 1 occurrences of 0.0047017 BTC output.
+There are 35 occurrences of 0.00524288 BTC output.
+There are 1 occurrences of 0.00580139 BTC output.
+There are 1 occurrences of 0.0062007 BTC output.
+There are 1 occurrences of 0.00708982 BTC output.
+There are 1 occurrences of 0.00719311 BTC output.
+There are 1 occurrences of 0.00741736 BTC output.
+There are 1 occurrences of 0.00789242 BTC output.
+There are 1 occurrences of 0.00799932 BTC output.
+There are 1 occurrences of 0.00813015 BTC output.
+There are 1 occurrences of 0.00869702 BTC output.
+There are 1 occurrences of 0.00996575 BTC output.
+There are 1 occurrences of 0.01027362 BTC output.
+There are 26 occurrences of 0.01048576 BTC output.
+There are 1 occurrences of 0.01318739 BTC output.
+There are 1 occurrences of 0.01339224 BTC output.
+There are 1 occurrences of 0.01445627 BTC output.
+There are 1 occurrences of 0.01462106 BTC output.
+There are 1 occurrences of 0.01607106 BTC output.
+There are 1 occurrences of 0.01707106 BTC output.
+There are 1 occurrences of 0.01757983 BTC output.
+There are 1 occurrences of 0.01762349 BTC output.
+There are 1 occurrences of 0.01849946 BTC output.
+There are 1 occurrences of 0.02002486 BTC output.
+There are 23 occurrences of 0.02097152 BTC output.
+There are 1 occurrences of 0.03033266 BTC output.
+There are 1 occurrences of 0.0393451 BTC output.
+There are 14 occurrences of 0.04194304 BTC output.
+There are 1 occurrences of 0.05653016 BTC output.
+There are 1 occurrences of 0.07940192 BTC output.
+There are 7 occurrences of 0.08388608 BTC output.
+There are 1 occurrences of 0.11869243 BTC output.
+There are 1 occurrences of 0.14460622 BTC output.
+There are 5 occurrences of 0.16777216 BTC output.
+There are 1 occurrences of 0.2086305 BTC output.
+There are 4 occurrences of 0.33554432 BTC output.
+There are 1 occurrences of 0.65823 BTC output.
+There are 3 occurrences of 0.67108864 BTC output.
+There are 2 occurrences of 1.34217728 BTC output.
+There are 2 occurrences of 2.68435456 BTC output.
+There are 1 occurrences of 4.72813231 BTC output.
+There are 1 occurrences of 5.36870912 BTC output.
+There are 1 occurrences of 10.73741824 BTC output.
+```
+
+#### Descending Powers of 2
+
+It would be more efficient to do descending powers of 2 mixing instead of ascending powers of 2 mixing like we did before. So we implemented this and here's what we noticed:
+
+- Now there are many dusts would had been created, but they don't matter, because they're cents and can just be swallowed by the miners.  
```

Comment:
```suggestion
- Now more dust coins would be created, but because of their low economic value, these coins are not created and instead the value is added to the miner fees.  
```